File: lib/models/todo.dart
class Todo {
  int id;
  String titulo;
  String descricao;
  bool concluido;

  Todo({
    this.id,
    this.titulo,
    this.descricao,
    this.concluido,
  });

  Todo.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    titulo = json['titulo'];
    descricao = json['descricao'];
    concluido = json['concluido'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['id'] = this.id ?? 0;
    data['titulo'] = this.titulo;
    data['descricao'] = this.descricao;
    data['concluido'] = this.concluido ?? false;
    return data;
  }
}

File: api/Models/Todo.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace TodoListApi.Models
{
    public class Todo
    {
        [Key]
        public int Id { get; set; }
        [Required]
        public string Titulo { get; set; }
        public string Descricao { get; set; }
        public bool Concluido { get; set; }
        public DateTime DataCriacao { get; set; }
        public DateTime? DataAtualizacao { get; set; }
    }
}
